pkg/controller/remotemachineset/gcpactuator.go

```diff
@@ -451,6 +463,32 @@ func getGCPImageID(masterMachine *machineapi.Machine, scheme *runtime.Scheme, lo
 	return imageID, nil
 }
 
+// Get the network information from existing machines on the remote cluster.
+func getNetwork(remoteMachineSets []machineapi.MachineSet,
+	scheme *runtime.Scheme, logger log.FieldLogger) (network, subnet string, err error) {
+	if len(remoteMachineSets) == 0 {
+		return
+	}
+
+	remoteMachineProviderSpec, err := decodeGCPMachineProviderSpec(
+		remoteMachineSets[0].Spec.Template.Spec.ProviderSpec.Value,
+		scheme,
+	)
+	if err != nil {
+		logger.WithError(err).Warn("cannot decode GCPMachineProviderSpec from master machine")
+		err = errors.Wrap(err, "cannot decode GCPMachineProviderSpec from master machine")
+		return
```

Comment:
```suggestion
		return "", "", errors.Wrap(err, "cannot decode GCPMachineProviderSpec from master machine")
```

using implicit returns creates confusion when reading the code as it doesn't become clear immediately that we are returning values compared to empty return..

Reply:
Done.